time_tracker_api/time_entries/time_entries_namespace.py

```diff
@@ -256,6 +256,18 @@ def post(self):
         return time_entries_dao.create(ns.payload), HTTPStatus.CREATED
 
 
```

Comment:
> why exactly? should not we write proper tests instead of adding this?
@enriquezrene  in this part of the code, Paul copied functionality of another endpoint to make it work. It works fine as it , It only seems to me that there may be extra logic that could be avoided. For that reason, I wanted to add a reminder to review that later.

We can enforce to remove that extra logic now or later, I am ok with both options.

Reply:
Lets do it now please